cmds.go

```diff
@@ -225,7 +218,7 @@ func exports(socket, certPath string) map[string]string {
 	if certPath == "" {
 		out["DOCKER_TLS_VERIFY"] = ""
 	} else {
-		out["DOCKER_TLS_VERIFY"] = "1"
+		out["DOCKER_TLS_VERIFY"] = "yes"
```

Comment:
to make it the same as in machine....

I'm basically taking the POV that if our code looks more like theirs, then its will be easier for people that might need to back-port changes.